python/ray/autoscaler/kubernetes/node_provider.py

```diff
@@ -85,3 +86,9 @@ def terminate_node(self, node_id):
     def terminate_nodes(self, node_ids):
         for node_id in node_ids:
             self.terminate_node(node_id)
+
+    def get_command_runner(self, log_prefix, node_id, provider, auth_config,
+                           cluster_name, process_runner, use_internal_ip):
+        return KubernetesCommandRunner(log_prefix, node_id, provider,
```

Comment:
provider == self right? So we could omit this.

Reply:
yes